Guided Hunting - Covid-19 Themed Threats.ipynb

```diff
@@ -39,42 +39,43 @@
    "metadata": {},
```

Comment:
<p>tldextract and IPWhois are in msticpy base - but no harm. Just FYI</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>